js/app.js

```diff
@@ -1,239 +1,174 @@
+new MakeImage('chair', 'img/chair.jpg', 0, 0),
+new MakeImage('cthulhu', 'img/cthulhu.jpg', 0, 0),
+new MakeImage('dogDuck', 'img/dogDuck.jpg', 0, 0),
+new MakeImage('dragon', 'img/dragon.jpg', 0, 0),
+new MakeImage('pen', 'img/pen.jpg', 0, 0),
+new MakeImage('petSweep', 'img/petSweep.jpg', 0, 0),
+new MakeImage('scissors', 'img/scissors.jpg', 0, 0),
+new MakeImage('shark', 'img/shark.jpg', 0, 0),
+new MakeImage('tauntaun', 'img/tauntaun.jpg', 0, 0),
+new MakeImage('unicorn', 'img/unicorn.jpg', 0, 0),
+new MakeImage('usb', 'img/usb.jpg', 0, 0),
+new MakeImage('waterCan', 'img/waterCan.jpg', 0, 0),
+new MakeImage('wineGlass', 'img/wineGlass.jpg', 0, 0);
+
+
+///functions//////////////////////
+
+//Limit Attempts///
+if (attempts === maxAttempts) {
+  chart();
+}
+//pull three from array///
 
-  var answer = event.target.getAttribute('id');
-  attempts++;
-//for i statement - constructor
-  if (answer === [0]) {
-    babySweep++;
-  } else if (answer === [1]) {
-    bag++;
-  } else if (answer === [2]) {
-    banana++;
-  } else if (answer === [3]) {
-    bathroom++;
-  } else if (answer === [4]) {
-    boots++;
-  } else if (answer === [5]) {
-    breakfast++;
-  } else if (answer === [6]) {
-    bubblegum++;
-  } else if (answer === [7]) {
-    chair++;
-  } else if (answer === [8]) {
-    cthulhu++;
-  } else if (answer === [9]) {
-    dogDuck++;
-  } else if (answer === [10]) {
-    dragon++;
-  } else if (answer === [11]) {
-    pen++;
-  } else if (answer === [12]) {
-    petSweep++;
-  } else if (answer === [13]) {
-    scissors++;
-  } else if (answer === [14]) {
-    shark++;
-  } else if (answer === [15]) {
-    tauntaun++;
-  } else if (answer === [16]) {
-    unicorn++;
-  } else if (answer === [17]) {
-    usb++;
-  } else if (answer === [18]) {
-    waterCan++;
-  } else if (answer === [19]) {
-    wineGlass++;
-  } else {
-    alert('please click on image');
-  }
 
-  if (attempts === maxAttempts) {
-    chart();
+function imgPath() {
+  var ip = MakeImage.setAttribute('src');
+  ip.value = product.path;
+  img.setAttributeNode(ip);
+}
+//render with math random///
+//check for dups///
+function render() {
+  var randomImg = function() {
+    img1 = Math.floor(Math.random() * imgPath.length);
+    img2 = Math.floor(Math.random() * imgPath.length);
+    img3 = Math.floor(Math.random() * imgPath.length);
+    newImg = [];
+    pushImg.push(img1, img2, img3);
+  };
+  while ( event.target.id === img1.name || event.target.id === img2.name || event.target.id === img3.name) {
+    randomImg();
   }
-});
-
-function ProductName(name, click, shown) {
-  this.name = name,
-  this.click = 0,
-  this.shown = 0
 }
 
-function generateRandomProduct() {
-  var index = Math.floor(Math.random() * productNames.length);
-  return productNames[index];
+//put three img into placing///
+function renderProductImage (products, imgAll) {
+  var x = document.getElementById('id'); document.createAttribute('src');
+  x.value = 'filepath';
```

Comment:
Your error:
Uncaught TypeError: Cannot set property 'value' of null
    at renderProductImage (app.js:92)
    at app.js:22